products/compute/terraform.yaml

```diff
@@ -356,6 +356,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         Because the API does not return the sensitive key value,
         we cannot confirm or reverse changes to a key outside of Terraform.
   DiskResourcePolicyAttachment: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      Adds existing resource policies to a disk. You can only add one policy
```

Comment:
```suggestion
      Adds existing resource policies to a zonal disk. You can only add one policy
```

Reply:
If this works for local SSD disks (is attaching a resource policy to one a thing that makes sense? I have no idea!), omit this suggestion.

Reply:
I actually had this written and then chose to omit it right before pushing :P I think since we specify that region disks aren't supported it's not needed.